querybook/webapp/components/SearchAndReplace/DraftJsSearchHighlighter.tsx

```diff
@@ -23,23 +23,26 @@ export const DraftJsSearchHighlighter: React.FC<{
     } = searchContext;
 
     const shouldHighlight = useMemo(
-        () => editor && searchResults.some((r) => r.cellId === cellId),
-        [searchResults, cellId, editor]
+        () =>
+            editorRef.current && searchResults.some((r) => r.cellId === cellId),
+        [searchResults, cellId, editorRef]
     );
     useEffect(() => {
-        if (editor) {
+        if (editorRef.current) {
             const decorators = [LinkDecorator];
             if (shouldHighlight) {
                 decorators.push(
                     makeSearchHighlightDecorator(searchString, searchOptions)
                 );
             }
 
-            editor.editorState = DraftJs.EditorState.set(editor.editorState, {
-                decorator: new DraftJs.CompositeDecorator(decorators),
-            });
+            editorRef.current.setEditorState(
+                DraftJs.EditorState.set(editorRef.current.getEditorState(), {
+                    decorator: new DraftJs.CompositeDecorator(decorators),
+                })
+            );
         }
-    }, [shouldHighlight, editor, searchString, searchOptions]);
+    }, [shouldHighlight, editorRef, searchString, searchOptions]);
```

Comment:
ref will not change, it does nothing in the deps array I guess?

Reply:
its just to suppress the hooks warning, since it doesn't change it shouldn't affect anything